### ISSUE TITLE
Local build using setup.py fails to compile

### ISSUE THREAD

**Issue by salotz** [msmbuilder/msmbuilder#942]
I need to set up a local version of msmbuilder for development. Using setup.py fails with `python setup.py install` gives `error: command 'gcc' failed with exit status 1`, full message in attached file [setup.py.txt](https://github.com/msmbuilder/msmbuilder/files/587963/setup.py.txt).
I first tried to just import but I get this.:
```
In [2]: import msmbuilder.msm as ms
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-3fbc4931e0ad> in <module>()
----> 1 import msmbuilder.msm as ms

/home/salotz/Dropbox/devel/msmbuilder/msmbuilder/msm/__init__.py in <module>()
      1 from __future__ import absolute_import
----> 2 from .core import *
      3 from .msm import MarkovStateModel
      4 from .ratematrix import ContinuousTimeMSM
      5 from .bayesmsm import BayesianMarkovStateModel

/home/salotz/Dropbox/devel/msmbuilder/msmbuilder/msm/core.py in <module>()
     14 from sklearn.utils import check_random_state
     15 
---> 16 from . import _ratematrix
     17 from ..utils import list_of_1d
     18 

ImportError: cannot import name '_ratematrix'
```

I am on Ubuntu 16.04 LTS, `Python 3.5.2 :: Anaconda custom (64-bit)`, and release 3.3.1 of MSMBuilder.

**Comment by rmcgibbo** [comment]
I believe 3.3.1 does not compile with GCC5, and that this issue has been fixed as of the latest release (3.6.1). Can you try with the newest release and report back of you have any problems?

**Comment by salotz** [comment]
All good, Thanks.
